### ISSUE TITLE
Focus is not working when Learner opens info side panel while in an activity

### ISSUE THREAD

**Issue by sairina** [learningequality/kolibri#9096]
## Observed behavior
When a learner opens up an activity from the Library and uses the keyboard to navigate to the "view information" icon to open the side panel, they are unable to use the keyboard to navigate the opened side panel.

Additional notes:
- in the code, these are the `FullScreenSidePanel` components in `LearnImmersiveLayout`
- there is an existing pattern with `FullScreenSidePanel` and the use of the `FocusTrap` component that can be applied in this issue
- You may run into issues with figuring out what the first element to focus in this particular side panel should be

## Errors and logs
![2022-02-09 23 00 19](https://user-images.githubusercontent.com/13563002/153354918-521eb4b6-86f0-4d32-a24d-c05030c5a65d.gif)

## Expected behavior
Focus behavior should match other instances of a full screen side panel

## User-facing consequences
It is an accessibility issue

## Steps to reproduce
1. As a learner, navigate to an activity
2. When the activity is opened, use the keyboard to navigate through the top app bar
3. Press "enter" when you reach the "view more information" icon on the furthest righthand side
4. Note that you cannot successfully navigate through the side panel with the keyboard

## Context
Kolibri v0.15.1

**Comment by marcellamaki** [comment]
Closed with https://github.com/learningequality/kolibri/pull/9408
